### PR TITLE
BETA: Register whsprr.is-a.dev

### PR DESCRIPTION
**Pull request by 0n4u**
Added `whsprr.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/whsprr.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "whsprr-png",
+        "email": "musicbotqn@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://whsprr-png.github.io/"
```
Comment (CuteDog5695) [inline domains/whsprr.json:7]: CNAME cant have https:// and /, remove those ones please

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/whsprr.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "whsprr-png",
+        "email": "musicbotqn@gmail.com"
+    },
+    "record": {
+        "CNAME": "whsprr-png.github.io"
```
Comment (CuteDog5695) [inline domains/whsprr.json:7]: page gives 404 error, but if you do have your website hosted with github then comment with the website link

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/whsprr.json (hunk 2 above)
Comment (whsprr-png) [reply domains/whsprr.json:7]: whsprr-png.github.io/bio

**Review by iostpa — APPROVED** [review]
now its fine

**Pull request merged**